src/core_read.cpp

```diff
@@ -40,8 +39,9 @@ CScript ParseScript(const std::string& s)
                 continue;
             mapOpNames[strName] = static_cast<opcodetype>(op);
             // Convenience: OP_ADD and just ADD are both recognized:
-            boost::algorithm::replace_first(strName, "OP_", "");
-            mapOpNames[strName] = static_cast<opcodetype>(op);
+            if (strName.rfind("OP_", 0) == 0) { // strName starts with "OP_"
```

Comment:
```suggestion
            if (strName.compare(0, 3, "OP_") == 0) { // strName starts with "OP_"
```

Reply:
Agreed that this is more readable, done.